third_party/2and3/cryptography/x509.pyi

```diff
@@ -296,6 +305,80 @@ class SubjectAlternativeName(ExtensionType):
     def __iter__(self) -> Generator[GeneralName, None, None]: ...
     def get_values_for_type(self, type: Type[GeneralName]) -> List[Any]: ...
 
+class AuthorityKeyIdentifier(ExtensionType):
+    key_identifier: bytes
```

Comment:
These are apparently read-only properties, so they should be annotated with `@property` (the current stub would not catch it if you try to assign to them).

This also goes for most (all?) other properties in this file.